setup.py

```diff
@@ -433,7 +433,7 @@ def run(self):
         extra_compile_args.append("/Z7")
         extra_link_args.append("/DEBUG")
 else:
-    extra_compile_args = ["-Werror"]
+    extra_compile_args = []
```

Comment:
We'll have to restore this before merging, but let's have a look and see how the arm build goes for now.

Reply:
I am confused that travis build has failed here whereas it has passed in https://travis-ci.org/github/ossdev07/pandas/builds/683838496 which has been triggered by the same commit.

Flag "-Werror" gives a compilation error while compiling external dependencies in arm64 job.

Reply:
I have added a commit by restoring "-Werror" flag, it is giving an error in pandas compilation.

**Error:**
pandas/_libs/src/ujson/lib/ultrajsonenc.c:397:1: error: ‘fastcall’ attribute directive ignored [-Werror=attributes]
 Buffer_AppendShortHexUnchecked(char *outputOffset, unsigned short value) {

I have checked pandas/_libs/src/ujson/lib/ultrajson.h file for attribute "FASTCALL_ATTR" definition which is throwing a warning and which returns as error due to -Werror flag. Could you suggest anything on resolving this issue?

Reply:
I don't know much about of that part of the code, sorry. But it must be something specific to arm, since it's working well with all other architectures.

Reply:
The macro definition for that is going to be in ultrajson.h . Might be some other platform check needed there

Reply:
My guess is has something to do with the definition of `__x86_64__` on the platform

Reply:
Thanks @WillAyd for the suggestion. I have added the definition as "if !defined __x86_64__ && !defined __aarch64__" in pandas/_libs/src/ujson/lib/ultrajson.h.

Reply:
@datapythonista / @WillAyd ,
Arm build is passing but 2 tests are failing in 2 amd64 jobs. But I think they are failing for network-specific reasons rather than adding arm jobs. 

Can we merge current changes or any more updates are required?